packages/dev/core/src/Materials/meshDebugPluginMaterial.ts

```diff
@@ -431,15 +428,15 @@ export class MeshDebugPluginMaterial extends MaterialPluginBase {
      * @param uniformBuffer Uniform buffer
      */
     public bindForSubMesh(uniformBuffer: UniformBuffer): void {
-        uniformBuffer.updateVector3("dbg_shadedDiffuseColor", this._options.shadedDiffuseColor);
-        uniformBuffer.updateFloat4("dbg_shadedSpecularColorPower", this._options.shadedSpecularColor.x, this._options.shadedSpecularColor.y, this._options.shadedSpecularColor.z, this._options.shadedSpecularPower);
+        uniformBuffer.updateFloat3("dbg_shadedDiffuseColor", this._options.shadedDiffuseColor.r, this._options.shadedDiffuseColor.g, this._options.shadedDiffuseColor.b);

```

Comment:
I think we can use `updateColor3`?

```suggestion
        uniformBuffer.updateColor3("dbg_shadedDiffuseColor", this._options.shadedDiffuseColor);
```